ui/hooks/useMultichainAccountsIntroModal.ts

```diff
@@ -33,17 +32,24 @@ export function useMultichainAccountsIntroModal(
   );
 
   const lastUpdatedAt = useAppSelector((state) => state.metamask.lastUpdatedAt);
-  const lastUpdatedFromVersion = useAppSelector(getLastUpdatedFromVersion);
+  const lastUpdatedFromVersion = useAppSelector(
+    (state) => state.metamask.previousAppVersion,
+  );
 
   useEffect(() => {
     // Only show modal on the main wallet/home route
     const isMainWalletArea = location.pathname === DEFAULT_ROUTE;
 
+    const parsedLastVersion = semverParse(lastUpdatedFromVersion);
+    // Strip prerelease versions as they just indicate build types.
+    const strippedLastVersion = parsedLastVersion
+      ? `${parsedLastVersion.major}.${parsedLastVersion.minor}.${parsedLastVersion.patch}`
+      : parsedLastVersion;
```

Comment:
For other reviewers: this will be nullish, so `strippedLastVersion` will effectively be `string | null`, thus the check below is still valid 👍